### ISSUE TITLE
Netedit segfaults when editing junction shape

### ISSUE THREAD

**Issue by jakobht** [eclipse/sumo#7326]
# To reproduce
2. Create new in netedit
3. Create an edge
4. Switch to inspection mode
5. Rightclick on one of the junctions and choose "set custom junction shape"
6. Click the edge

Netedit will segfault

# System
```bash
$ sumo --version
Eclipse SUMO Version 1.6.0
 Build features: Linux-5.7.9-arch1-1 x86_64 GNU 10.1.0 Release Proj GUI SWIG GDAL FFmpeg OSG GL2PS
 Copyright (C) 2001-2020 German Aerospace Center (DLR) and others; https://sumo.dlr.de

Eclipse SUMO Version 1.6.0 is part of SUMO.
This program and the accompanying materials
are made available under the terms of the Eclipse Public License v2.0
which accompanies this distribution, and is available at
http://www.eclipse.org/legal/epl-v20.html
SPDX-License-Identifier: EPL-2.0
```

OS: Arch linux with gnome 

I installed sumo from [AUR](https://aur.archlinux.org/packages/sumo/)

**Comment by palvarezlopez** [comment]
@jakobht thanks for report it

**Comment by jakobht** [comment]
I just downgraded to 1.5 and here it is not an issue

**Comment by palvarezlopez** [comment]
@jakobht This bug was already fixed in #6945, and we're going to include in the next 1.7 release (20 September). If you want, you can use the daily Nightly Snapshots version

https://sumo.dlr.de/docs/Downloads.php
